package-lock.json

```diff
@@ -1,8 +1,4540 @@
 {
   "name": "dapr-client",
-  "version": "0.9.0-preview01",
-  "lockfileVersion": 1,
+  "version": "0.12.0",
```

Comment:
It is still preview at this point. We remove the RC on the day of release. On the other hand, given the scope of this SDK, it might be OK to skip and release the minor version right away.